nibabel/nifti1.py

```diff
@@ -9,6 +9,12 @@
 ''' Read / write access to NIfTI1 image format
 
 NIfTI1 format defined at http://nifti.nimh.nih.gov/nifti-1/
+
```

Comment:
Can we make this two separate strings, so the user doesn't get this testsetup in the docstring help?